src/lib/Repository/Validator/TargetContentValidator.php

```diff
@@ -13,7 +13,7 @@
 use Ibexa\Core\FieldType\ValidationError;
 
 /**
- * Validator for checking existence of content and its content type.
+ * Validator for checking existence of content and its Content type.
```

Comment:
Seems that one was correct initially.
```suggestion
 * Validator for checking existence of content and its content type.
```